tests/resources/Harbor-Pages/Configuration.robot

```diff
@@ -276,3 +276,18 @@ Delete A Label
     Capture Page Screenshot
     Click Element  xpath=//clr-modal//div//button[contains(.,'DELETE')]
     Wait Until Page Contains Element  //clr-tab-content//div[contains(.,'${labelname}')]/../div/clr-icon[@shape="success-standard"]
+
+## Garbage Collection	
+Switch To Garbage Collection
+    Sleep  1
+    Click Element  xpath=${configuration_xpath}
+    Click Element  xpath=${garbage_collection_xpath}
+
+Click GC Now
+    Sleep  1
+    Click Element  xpath=${gc_now_xpath}
+    Sleep  5
```

Comment:
Seems the`sleep 5` here is useless as the `Wait Until Page Contains  Finished` is used in the following case.